src/GitHub.VisualStudio/Helpers/ActiveDocumentSnapshot.cs

```diff
@@ -19,7 +19,7 @@ public class ActiveDocumentSnapshot : IActiveDocumentSnapshot
         public ActiveDocumentSnapshot([Import(typeof(SVsServiceProvider))] IServiceProvider serviceProvider)
         {
             StartLine = EndLine = -1;
-            Name = Services.Dte2?.ActiveDocument?.FullName;
+            Name = Services.Dte2?.ActiveDocument?.ProjectItem.Name;
```

Comment:
When you open a file in VS without a solution open, VS creates a temporary solution file so  `ProjectItem` still exists, but the link that's generated points to the latest commit of the git repo that's currently in the context. Looks like a bug there, it probably should generate a link if the solution isn't open, I'll put that in another issue.